### PR TITLE
export_incidents_to_csv_batch fix example

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/demisto-py/issues/66

### CODE REVIEW

File: docs/DefaultApi.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ Method | HTTP request | Description
 [**edit_ad_hoc_task**](DefaultApi.md#edit_ad_hoc_task) | **POST** /inv-playbook/task/edit/{investigationId} | Edit ad-hoc task
 [**entry_export_artifact**](DefaultApi.md#entry_export_artifact) | **POST** /entry/exportArtifact | Export Artifact
 [**execute_report**](DefaultApi.md#execute_report) | **POST** /report/{id}/{requestId}/execute | Execute report
-[**export_incidents_to_csv_batch**](DefaultApi.md#export_incidents_to_csv_batch) | **POST** /incident/batch/exportToCsv | Batch export incidents to csv
```
Comment (Itay4) [inline docs/DefaultApi.md:29]: why did you remove it?

Comment (jochman) [reply docs/DefaultApi.md]: by mistake. fixed

**Review by Itay4 — APPROVED** [review]
